packages/client/src/runtime/core/engines/common/Engine.ts

```diff
@@ -147,6 +147,13 @@ export interface EngineConfig {
    * in the current working directory. This usually means it has been bundled.
    */
   isBundled?: boolean
+
+  /**
+   * Gets the raw wasm module for the query engine. This configurations is
+   * generated specifically for each type of client, eg. Node.js client and Edge
+   * clients will have different implementations.
```

Comment:
Maybe also worth mentioning that is is a callback on *purpose*. We only load the wasm if we need it.

Reply:
```suggestion
   * Gets the raw wasm module for the query engine. This configurations is
   * generated specifically for each type of client, eg. Node.js client and Edge
   * clients will have different implementations.
   * Note: that is is a callback on purpose. We only load the wasm if we need it.
```